packages/qri-node/lib/qri-node.js

```diff
@@ -1,25 +1,41 @@
 const fs = require('fs-extra')
-const { spawn } = require('child_process');
+const { spawnSync } = require('child_process');
 
 module.exports = {
+  /**
+  * 'initialize' creates a new dataset, links it to the current directory,
+  * and creates starter files for the dataset's components
+  * @param {string} dirPath - The path to create the new dataset in
+  */
   init: (dirPath) => {
     console.log(`initializing Qri Dataset in ${dirPath}`)
     // make an empty dir
     fs.emptyDirSync(dirPath)
```

Comment:
This function is really dangerous, it will delete everything inside the given directory. So if a user runs `qri.init("~/")` it will remove everything in their home directory. It does not, as the comment says, merely create an empty directory.

See: https://github.com/jprichardson/node-fs-extra/blob/master/docs/emptyDir-sync.md

The underlying `qri init` command will instead work okay if the directory is empty, unless it contains files that conflict with known filenames we care about. It also has an option to create an empty directory within a path, as a separate argument. For example: `qri init my_empty_dir` will create "my_empty_dir". Perhaps that would be a better interface, one function for where to place the dataset, with an optional directory name to create if desired.

Reply:
I'll chat with you on discord about the best solution to this.